world/petitions/models.py

```diff
@@ -21,19 +108,38 @@ class BrokeredSale(SharedMemoryModel):
     CRAFTING_MATERIALS = 4
     SALE = 0
     PURCHASE = 1
-    OFFERING_TYPES = ((ACTION_POINTS, "Action Points"), (ECONOMIC, "Economic Resources"), (SOCIAL, "Social Resources"),
-                      (MILITARY, "Military Resources"), (CRAFTING_MATERIALS, "Crafting Materials"))
-    RESOURCE_TYPES = ((ECONOMIC, "economic"), (SOCIAL, "social"), (MILITARY, "military"))
+    OFFERING_TYPES = (
+        (ACTION_POINTS,
+         "Action Points"),
+        (ECONOMIC,
+         "Economic Resources"),
+        (SOCIAL,
+         "Social Resources"),
+        (MILITARY,
+         "Military Resources"),
+        (CRAFTING_MATERIALS,
+         "Crafting Materials"))
+    RESOURCE_TYPES = ((ECONOMIC, "economic"),
+                      (SOCIAL, "social"), (MILITARY, "military"))
     BROKER_TYPES = ((PURCHASE, "Purchase"), (SALE, "Sale"))
-    owner = models.ForeignKey("dominion.PlayerOrNpc", related_name="brokered_sales")
-    sale_type = models.PositiveSmallIntegerField(default=ACTION_POINTS, choices=OFFERING_TYPES)
+    owner = models.ForeignKey(
+        "dominion.PlayerOrNpc",
+        related_name="brokered_sales")
+    sale_type = models.PositiveSmallIntegerField(
+        default=ACTION_POINTS, choices=OFFERING_TYPES)
     amount = models.PositiveIntegerField(default=0)
     price = models.PositiveIntegerField(default=0)
-    buyers = models.ManyToManyField("dominion.PlayerOrNpc", related_name="brokered_purchases",
-                                    through="PurchasedAmount")
-    crafting_material_type = models.ForeignKey("dominion.CraftingMaterialType", null=True, blank=True,
-                                               on_delete=models.CASCADE)
-    broker_type = models.PositiveSmallIntegerField(default=SALE, choices=BROKER_TYPES)
+    buyers = models.ManyToManyField(
+        "dominion.PlayerOrNpc",
+        related_name="brokered_purchases",
+        through="PurchasedAmount")
+    crafting_material_type = models.ForeignKey(
+        "dominion.CraftingMaterialType",
+        null=True,
+        blank=True,
+        on_delete=models.CASCADE)
+    broker_type = models.PositiveSmallIntegerField(
+        default=SALE, choices=BROKER_TYPES)
```

Comment:
As I mentioned in the other code review, I apologize for not mentioning it, but the one PEP8 bit I don't adhere to is the 80 character per line limit, preferring 120. I much prefer horizontal placement of arguments for functions/classes so it's easier to tell what's going on when scrolling through code.